stable/haproxy/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "2.0.1"
+description: A Helm chart for HAProxy
+name: haproxy
+version: 0.1.0
```

Comment:
Not requiring this to change: shouldn't a `stable` chart at least be v `1.0.0` ? 🤔